logging-api/src/main/java/com/cisco/oss/foundation/logging/transactions/SchedulerLogger.java

```diff
@@ -47,6 +56,20 @@ public static void success(String schedulerResult) {
 
     }
 
+    public static void successAsync(SchedulerLogger schedulerLogger) {
+
+        FlowContextFactory.deserializeNativeFlowContext(TransactionLogger.getFlowContextAsync(schedulerLogger));
+        schedulerLogger.successInstance();
+
+    }
+
+    public void successAsync() {
+
+        FlowContextFactory.deserializeNativeFlowContext(this.flowContext);
+        this.successInstance();
```

Comment:
Why will it be called twice? There's one static function - I put this in so that the successAsync api in TaskL and HTTPSpringL would exist here too, and added another not-static successAsync api which seems to make more sense for an instance based logger design, as we discussed previously.